watsononlinestore/watson_online_store.py

```diff
@@ -73,6 +73,8 @@ def __init__(self, bot_id, slack_client,
             'DISCOVERY_ENVIRONMENT_ID')
         self.discovery_collection_id = os.environ.get(
             'DISCOVERY_COLLECTION_ID')
+        self.discovery_score_filter = float(os.environ.get(
```

Comment:
This needs a try catch in because a non-convertible string raises an exception.  Unless that error looks good enough as-is, but I doubt that.